format/index.md

```diff
@@ -300,6 +300,11 @@ pagination links, as described below.
 If a link object refers to resource objects included in the same compound
 document, it **MUST** include resource linkage to those resource objects.
 
+> Note: a resource may actually be referring to another resource returned 
+in the same compound document, but this reference may be omitted 
+from the link object of the parent resource because of sparse fieldset 
+restrictions.
+
```

Comment:
I agree with all of you, @dgeb, @ethanresnick and @bintoro, but at the same time I'd like to defend my suggestion. The reason I made it a note was that it did not actually add anything normative to the spec, but just clarified the (indeed obvious) point that, if sparse fieldset requires to drop some relationship, it will not be listed in `links`. There are many notes in the spec which are of purely explanatory nature, to help the reader of the spec to put pieces together, and this one was meant to be of that kind. If you think it is insultingly unnecessary, I do not mind removing it.